packages/core/src/internal/motion/utils.ts

```diff
@@ -355,12 +355,12 @@ export function sizeDimensionKeyframes(
   }
 
   return animationKeyframes.map((kf: Keyframe) => {
-    if (kf?.hasOwnProperty('height') && isPrefixedBy(kf?.height?.toString() || '', 'from:')) {
+    if (Object.prototype.hasOwnProperty.call(kf, 'height') && isPrefixedBy(kf?.height?.toString() || '', 'from:')) {
```

Comment:
Eslint did not like us calling hasOwnProperty, I forget the exact rule. This is the proper way to call methods on objects.